docs/sample-v2-addon/src/test-support/components/ui/page.ts

```diff
@@ -0,0 +1,7 @@
+import styles from '../../../components/ui/page.css';
+
+type LocalClass = keyof typeof styles;
+
+export function getClassForUiPage(localClass: LocalClass): string {
+  return styles[localClass];
+}
```

Comment:
We see the correct type for `localClass`.

<img width="800" alt="" src="https://github.com/ijlee2/embroider-css-modules/assets/16869656/f6e39ca8-50ab-4a68-8bce-ed9417e22d3b">